plugins/scaffolder-backend/src/scaffolder/actions/builtin/fetch/template.ts

```diff
@@ -256,3 +257,14 @@ export function createFetchTemplateAction(options: {
     },
   });
 }
+
+function containsSkippedContent(localOutputPath: string): boolean {
+  // if the path starts with / means that the root directory has been skipped
+  // if the path is empty means that there is a file skipped in the root
+  // if the path includes // means that there is a subdirectory skipped
+  return (
+    localOutputPath.startsWith('/') ||
+    localOutputPath === '' ||
+    localOutputPath.includes('//')
```

Comment:
Hm. I think we have to amend these checks.

For one, I think we have to check for both forward and backward slashes, so that we capture the same type of problem on windows machines. Then there's the special thing where some windows paths can start with `\\` right? Like uniform network paths or something.

Soo ... how to best do this?

We could for sure check https://nodejs.org/api/path.html#pathisabsolutepath , and then maybe check for ```.slice(1).includes(`${path.sep}${path.sep}`)``` or something like that?

Do we also need to check for `.endsWith(path.sep)`? If the template string was like `some/root/${ctx.filename}`

Reply:
I have checked it, if we check for `.endsWith(path.sep)` we break the tests to ensure that empty directories are created